### PR TITLE
Cap 108 archive endpoints

### PR DESCRIPTION
**Pull request by FinchHop**

### CODE REVIEW

File: server.py [hunk 1]
```diff
@@ -82,7 +119,7 @@ def load_data():
                 result = {'message': error_msg}
                 return make_response(jsonify(result), 404)
             try:
-                row_data = IntakeRow(request.get_json(force=True)).value_array()
+                row_data = (request.get_json(force=True)).value_array()
```
Comment (anhaley) [inline server.py:122]: Was this removal intentional?

File: server.py [hunk 2]
```diff
@@ -27,34 +29,69 @@ def allowed_file(filename):
...
+        POST /list to process specific query
+    Returns ({}): JSON object of table data
+    """
+    valid, msg = checkTableRouting(request.args.get('table'), restrictedTables)
+    if not valid:
+        return make_response(jsonify(msg), 404)
+    if request.method == 'POST':
+        return table_listing_post(request)
+    if request.method == 'GET':
+        return table_listing_get(request)
```
Comment (anhaley) [inline server.py:76]: The 'admin' parameter should be part of the table_listing_post/get functions, not the server routes. Have fetch_admin_data pass 'admin=True' to those functions, while fetch_data doesn't need to pass anything and the table listing functions have a default parameter of 'admin=False'. That way, requests that come via fetch_admin_data can be distinguished from requests that come via fetch_data. Do the check as to whether the table is in the appropriate list there, based on the value of 'admin'.

File: server.py [hunk 3]
```diff
@@ -16,6 +16,8 @@
 # Utilizes CORS with the list of origin strings and regex expressions to validate resource requests.
 CORS(app, origins=origin_list)
 
+restrictedTables = ['archive', 'txn_history']
+nonRestrictedTables = ['intake', 'metadata', 'violations', 'records']
```
Comment (anhaley) [inline server.py:20]: The /adminlist endpoint shouldn't be prevented from fetching non-restricted tables. You can either concatenate both of these lists for /adminlist's check (creating a master list) or just skip the check for /adminlist and let that route query whatever table it wants to

Comment (FinchHop) [reply server.py:20]: Fixed - deleted table checking

File: server.py (hunk 1 above)
Comment (FinchHop) [reply server.py:122]: Nope! Fixed this

File: server.py (hunk 2 above)
Comment (FinchHop) [reply server.py:76]: I think this is fixed now. Based off the other comment, checking the table is totally deleted now. Should I reinstate this within the "post"/"get" functions?

Comment (anhaley) [reply server.py]: Yes, we still want to check whether the request came from an admin-validated route. So add back the checkTableRouting functionality, but make a distinction between the tables available to a non-admin request and an admin request. In the table_listing_get/post functions, use the value of 'admin' to verify that the requested table is in the right list. 

You could do this one of two ways: first, you could have the two lists of tables you had in a previous commit, and use nonRestrictedTables as the "acceptable" list for the non-admin path and restrictedTables + nonRestrictedTables as the acceptable list for the admin path.

A better approach though would be to leverage the fact that the driver already has a get_table_list function to create a list of tables that are active in the database. This could serve as the acceptable list for the admin route. You could then have a short list of "admin-only" tables (for now, transactions and archive) and remove these tables from the acceptable list for the non-admin route. This approach requires some extra work and might involve moving the admin-vs-non-admin logic to the driver, but has the benefit that get_table_list doesn't rely on hard-coded lists of tables that can become outdated. In general, we want to avoid this kind of hardcoded list wherever possible, since it increases the need for maintenance and the potential for bugs as the project develops over time.

**Comment by FinchHop** [comment]
Checking for admin availability with requested tables
